Updated all VI descriptions in the scripting API

oscarfonloz opened the pull request:
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device/blob/main/CONTRIBUTING.md).

TODO: Check the above box with an 'x' indicating you've read and followed [CONTRIBUTING.md](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Adding VI descriptions to all VIs and CTL files in the scripting API

### Why should this Pull Request be merged?

We require of documentation in the VIs to ship a properly built API.

### What testing has been done?

No functional changes were made to the VIs. All unit tests pass.

rtzoeller commented:
Overall thoughts:
- I'm not sure it makes sense to provide descriptions for VIs which are not on the palettes. It's been my experience that descriptions are almost never checked for when updating VIs which aren't in the public API surface, which can lead to them being misleading or outright wrong as the codebase evolves. @buckd I'd like your thoughts on this as well.
- IMO the words "private data" should not be used. Descriptions can typically be more concise without them, and they leak implementation details needlessly.
	- For example, "Writes the conversion time to the module private data." could be "Writes the conversion time to the module."

Individual items:
- [ ] `Create Counter Driven Output.vi` and `Read Counter Driven Output.vi` are missing documentation.

buckd commented:
> * I'm not sure it makes sense to provide descriptions for VIs which are not on the palettes. It's been my experience that descriptions are almost never checked for when updating VIs which aren't in the public API surface, which can lead to them being misleading or outright wrong as the codebase evolves. @buckd I'd like your thoughts on this as well.

If it's not on the palette, it doesn't really need a description. That's an approach I've seen many times on VIs shipped by LabVIEW.

oscarfonloz commented:
Waiting on [PR 152](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device/pull/152) to go in because it removes two VIs that are duplicate and show as without description.

oscarfonloz commented:
> Overall thoughts:
> 
> * I'm not sure it makes sense to provide descriptions for VIs which are not on the palettes. It's been my experience that descriptions are almost never checked for when updating VIs which aren't in the public API surface, which can lead to them being misleading or outright wrong as the codebase evolves. @buckd I'd like your thoughts on this as well.

Per @buckd's comments, we should remove them. Currently the PR stands with all VIs including a description. I will work with @debryant to remove those that are internal. @rtzoeller let us know in Teams if you're able to get the extension of the documentation tool to know which VIs are in the palettes (right now it seems that just with Deborah's approach we should be fine. I leave to your discretion if you prefer to merge as-is, or wait for us to work on removing those descriptions.

> * IMO the words "private data" should not be used. Descriptions can typically be more concise without them, and they leak implementation details needlessly.
>   
>   * For example, "Writes the conversion time to the module private data." could be "Writes the conversion time to the module."

Makes a lot of sense. I removed all mentions of private data from all VIs.

> Individual items:
> 
> * [x]  `Create Counter Driven Output.vi` and `Read Counter Driven Output.vi` are missing documentation.

Fixed with PR 152

rtzoeller commented:
My only feedback is that `Get Slot.vim`'s help is misleading.

You aren't _required_ to provide the class constant for the module type. It's optional, but useful if you know the type of module you are retrieving.

@debryant can you fix this in your high level pass? I am going to submit this PR without this fix.
